test/Unit/Models/classes/class.OutcomeVariableTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoResultServer\test\Unit\Models\classes;
```

Comment:
Why did you rename `models` to `Models`? Now it no longer matches the tested classes

Reply:
Reverted